server/api/tls/index.js

```diff
@@ -12,60 +12,12 @@ import {Router as router} from 'express';
 const tlsApi = router();
 const bigQueryClient = new BigQuery();
 
-// TODO (SofiaVega) delete this (this route is the same as the one below)
-tlsApi.get('/tls-requests', async (req, res) => {
-  const version = req.query.version;
-  const year = req.query.year;
-  const month = req.query.month;
-  let table = 'httparchive.sample_data.requests_desktop_10k';
-  if (year != null && month != null) {
-    table = 'httparchive.requests.' + year + '_' + month + '_01_desktop';
-  }
-  let queryDescription = 'Number of requests per TLS version';
-  let sqlQuery =
-    `SELECT tls, COUNT(tls) as requests
-  FROM
-  (SELECT JSON_EXTRACT(payload, '$._securityDetails.protocol') as tls
-  FROM 
-  ` +
-    table +
-    `
-  where url like ("https%"))
-  GROUP BY tls`;
-  if (version != null) {
-    queryDescription = 'Number of requests that use TLS ' + version;
-    sqlQuery =
-      `SELECT COUNT(url) as requests
-    FROM ` +
-      table +
-      `
-    WHERE url like ("https%") AND 
-    JSON_EXTRACT(payload, '$._securityDetails.protocol') 
-    like "%TLS%` +
-      version +
-      `%"`;
-  }
-
-  const options = {
-    query: sqlQuery,
-    location: 'US',
-  };
-
-  const [rows] = await bigQueryClient.query(options);
-
-  res.json({
-    description: queryDescription,
-    table: table,
-    result: rows,
-  });
-});
-
 tlsApi.get('/tls-version', async (req, res) => {
   const query = queries.TlsRequests;
-  let table = req.query.table;
+  let table = 'httparchive.smaller_sample_requests';
```

Comment:
Are you going to use only the smaller sample table and replace it for the bigger table?

Reply:
I just changed this to the bigger sample table which is the default (before the user chooses a year)